src/Orc.DataAccess/Database/DataSourceSchemaProviders/MsSqlDataSourceSchemaProvider.cs

```diff
@@ -32,10 +30,18 @@ public DbDataSourceSchema GetSchema(DbConnectionString connectionString)
             using var dataReader = command.ExecuteReader();
             while (dataReader.Read())
             {
-                databases.Add(dataReader[0].ToString());
+                var readValue = dataReader[0].ToString();
```

Comment:
could dataREader[0] return null? In such case, we should use dataReader[0]?.ToString()